dex-auth/dex-crds/overlays/github/params.env

```diff
@@ -0,0 +1,11 @@
+# Dex Server Parameters (some params are shared with client)
+namespace=auth
+issuer=https://dex.example.com:32000
+client_id=kubeflow-oidc-authservice
+oidc_redirect_uris=['http://login.example.org:5555/callback/onprem-cluster']
+application_secret=pUBnBOY80SnXgjibTYM9ZWNzY2xreNGQok
+github_client_id=
+github_client_secret=
+github_hostname=
+github_org_name=
+github_use_login_as_id=false
```

Comment:
nit: add a new line for EOF?

Reply:
see https://github.com/kubeflow/manifests/pull/1202/commits/57166db1933d219273e4d0f8891a22b653ab3acb